typescript/sdk/src/core/HyperlaneCoreDeployer.ts

```diff
@@ -32,6 +38,20 @@ export class HyperlaneCoreDeployer extends HyperlaneDeployer<
     });
   }
 
+  async deployTimelock(
+    chain: ChainName,
+    delay: number,
+    owner: types.Address,
+  ): Promise<TimelockController> {
+    const timelock = await this.deployContract(
+      chain,
+      'timelockController',
+      // delay, [proposers], [executors], admin
+      [delay, [owner], [owner], ethers.constants.AddressZero],
```

Comment:
Why is address zero the admin? (just curious)

Reply:
dumb API https://github.com/OpenZeppelin/openzeppelin-contracts/blob/874c2d3c02ec1bce6af9a30bc828d3fe2079136b/contracts/governance/TimelockController.sol#L107

Reply:
I will add a comment